scripts/quests/crystalWar/A_Little_Knowledge.lua

```diff
@@ -0,0 +1,143 @@
+-----------------------------------
+-- A Little Knowledge
+-----------------------------------
+-- !addquest 7 6
+-- Erlene      : !pos 376.936 -39.999 17.914 175
+-----------------------------------
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.CRYSTAL_WAR, xi.quest.id.crystalWar.A_LITTLE_KNOWLEDGE)
+
+quest.reward =
+{
+    ki = xi.ki.GRIMOIRE,
+    title = xi.title.SCHULTZ_SCHOLAR,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and player:getMainLvl() >= xi.settings.main.ADVANCED_JOB_LEVEL
```

Comment:
Even though scripts/globals/settings is empty, we've been using the pattern of still adding the require

Reply:
I would like to but...
![image](https://github.com/LandSandBoat/server/assets/1618002/78dd56e4-d169-4fe8-b89e-995197c7292f)

Reply:
Yep, that's on me after that pattern was changed recently and didn't update this issue.  You're good here 👍